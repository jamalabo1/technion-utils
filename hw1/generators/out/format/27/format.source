A W 539 1
C W 958 2
C W 100 3
C Y 267 4
C Z 747 5
D X 707 6
A X 673 7
D Z 429 8
C X 932 9
B W 202 10
A Y 913 11
D W 138 12
B W 66 13
D Y 692 14
B X 485 15
A W 1 16
C Z 548 17
D Y 650 18
B W 666 19
B Z 173 20
D X 278 21
C W 916 22
B Z 375 23
D X 77 24
C X 503 25
B W 759 26
D W 222 27
A X 934 28
C Z 689 29
B Y 74 30
C X 952 31
A Y 259 32
D X 167 33
D Y 984 34
B Z 777 35
A W 755 36
D Z 62 37
C W 83 38
A X 498 39
D Y 390 40
C Y 393 41
A Y 67 42
B Y 927 43
D Z 492 44
D W 546 45
B Z 899 46
B W 332 47
A Y 487 48
B Z 793 49
D Z 159 50
A X 730 51
A Y 555 52
D Y 938 53
D X 492 54
A Z 24 55
D W 552 56
A X 175 57
D X 295 58
C X 186 59
D X 600 60
D Z 703 61
C Z 342 62
B Z 206 63
A Y 302 64
C Z 448 65
A Z 921 66
A W 344 67
B X 913 68