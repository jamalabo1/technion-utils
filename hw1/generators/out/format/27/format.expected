Blockchain Info:
1
Sender Name: B
Receiver Name: X
Transaction Value: 913
Transaction timestamp: 68
2
Sender Name: A
Receiver Name: W
Transaction Value: 344
Transaction timestamp: 67
3
Sender Name: A
Receiver Name: Z
Transaction Value: 921
Transaction timestamp: 66
4
Sender Name: C
Receiver Name: Z
Transaction Value: 448
Transaction timestamp: 65
5
Sender Name: A
Receiver Name: Y
Transaction Value: 302
Transaction timestamp: 64
6
Sender Name: B
Receiver Name: Z
Transaction Value: 206
Transaction timestamp: 63
7
Sender Name: C
Receiver Name: Z
Transaction Value: 342
Transaction timestamp: 62
8
Sender Name: D
Receiver Name: Z
Transaction Value: 703
Transaction timestamp: 61
9
Sender Name: D
Receiver Name: X
Transaction Value: 600
Transaction timestamp: 60
10
Sender Name: C
Receiver Name: X
Transaction Value: 186
Transaction timestamp: 59
11
Sender Name: D
Receiver Name: X
Transaction Value: 295
Transaction timestamp: 58
12
Sender Name: A
Receiver Name: X
Transaction Value: 175
Transaction timestamp: 57
13
Sender Name: D
Receiver Name: W
Transaction Value: 552
Transaction timestamp: 56
14
Sender Name: A
Receiver Name: Z
Transaction Value: 24
Transaction timestamp: 55
15
Sender Name: D
Receiver Name: X
Transaction Value: 492
Transaction timestamp: 54
16
Sender Name: D
Receiver Name: Y
Transaction Value: 938
Transaction timestamp: 53
17
Sender Name: A
Receiver Name: Y
Transaction Value: 555
Transaction timestamp: 52
18
Sender Name: A
Receiver Name: X
Transaction Value: 730
Transaction timestamp: 51
19
Sender Name: D
Receiver Name: Z
Transaction Value: 159
Transaction timestamp: 50
20
Sender Name: B
Receiver Name: Z
Transaction Value: 793
Transaction timestamp: 49
21
Sender Name: A
Receiver Name: Y
Transaction Value: 487
Transaction timestamp: 48
22
Sender Name: B
Receiver Name: W
Transaction Value: 332
Transaction timestamp: 47
23
Sender Name: B
Receiver Name: Z
Transaction Value: 899
Transaction timestamp: 46
24
Sender Name: D
Receiver Name: W
Transaction Value: 546
Transaction timestamp: 45
25
Sender Name: D
Receiver Name: Z
Transaction Value: 492
Transaction timestamp: 44
26
Sender Name: B
Receiver Name: Y
Transaction Value: 927
Transaction timestamp: 43
27
Sender Name: A
Receiver Name: Y
Transaction Value: 67
Transaction timestamp: 42
28
Sender Name: C
Receiver Name: Y
Transaction Value: 393
Transaction timestamp: 41
29
Sender Name: D
Receiver Name: Y
Transaction Value: 390
Transaction timestamp: 40
30
Sender Name: A
Receiver Name: X
Transaction Value: 498
Transaction timestamp: 39
31
Sender Name: C
Receiver Name: W
Transaction Value: 83
Transaction timestamp: 38
32
Sender Name: D
Receiver Name: Z
Transaction Value: 62
Transaction timestamp: 37
33
Sender Name: A
Receiver Name: W
Transaction Value: 755
Transaction timestamp: 36
34
Sender Name: B
Receiver Name: Z
Transaction Value: 777
Transaction timestamp: 35
35
Sender Name: D
Receiver Name: Y
Transaction Value: 984
Transaction timestamp: 34
36
Sender Name: D
Receiver Name: X
Transaction Value: 167
Transaction timestamp: 33
37
Sender Name: A
Receiver Name: Y
Transaction Value: 259
Transaction timestamp: 32
38
Sender Name: C
Receiver Name: X
Transaction Value: 952
Transaction timestamp: 31
39
Sender Name: B
Receiver Name: Y
Transaction Value: 74
Transaction timestamp: 30
40
Sender Name: C
Receiver Name: Z
Transaction Value: 689
Transaction timestamp: 29
41
Sender Name: A
Receiver Name: X
Transaction Value: 934
Transaction timestamp: 28
42
Sender Name: D
Receiver Name: W
Transaction Value: 222
Transaction timestamp: 27
43
Sender Name: B
Receiver Name: W
Transaction Value: 759
Transaction timestamp: 26
44
Sender Name: C
Receiver Name: X
Transaction Value: 503
Transaction timestamp: 25
45
Sender Name: D
Receiver Name: X
Transaction Value: 77
Transaction timestamp: 24
46
Sender Name: B
Receiver Name: Z
Transaction Value: 375
Transaction timestamp: 23
47
Sender Name: C
Receiver Name: W
Transaction Value: 916
Transaction timestamp: 22
48
Sender Name: D
Receiver Name: X
Transaction Value: 278
Transaction timestamp: 21
49
Sender Name: B
Receiver Name: Z
Transaction Value: 173
Transaction timestamp: 20
50
Sender Name: B
Receiver Name: W
Transaction Value: 666
Transaction timestamp: 19
51
Sender Name: D
Receiver Name: Y
Transaction Value: 650
Transaction timestamp: 18
52
Sender Name: C
Receiver Name: Z
Transaction Value: 548
Transaction timestamp: 17
53
Sender Name: A
Receiver Name: W
Transaction Value: 1
Transaction timestamp: 16
54
Sender Name: B
Receiver Name: X
Transaction Value: 485
Transaction timestamp: 15
55
Sender Name: D
Receiver Name: Y
Transaction Value: 692
Transaction timestamp: 14
56
Sender Name: B
Receiver Name: W
Transaction Value: 66
Transaction timestamp: 13
57
Sender Name: D
Receiver Name: W
Transaction Value: 138
Transaction timestamp: 12
58
Sender Name: A
Receiver Name: Y
Transaction Value: 913
Transaction timestamp: 11
59
Sender Name: B
Receiver Name: W
Transaction Value: 202
Transaction timestamp: 10
60
Sender Name: C
Receiver Name: X
Transaction Value: 932
Transaction timestamp: 9
61
Sender Name: D
Receiver Name: Z
Transaction Value: 429
Transaction timestamp: 8
62
Sender Name: A
Receiver Name: X
Transaction Value: 673
Transaction timestamp: 7
63
Sender Name: D
Receiver Name: X
Transaction Value: 707
Transaction timestamp: 6
64
Sender Name: C
Receiver Name: Z
Transaction Value: 747
Transaction timestamp: 5
65
Sender Name: C
Receiver Name: Y
Transaction Value: 267
Transaction timestamp: 4
66
Sender Name: C
Receiver Name: W
Transaction Value: 100
Transaction timestamp: 3
67
Sender Name: C
Receiver Name: W
Transaction Value: 958
Transaction timestamp: 2
68
Sender Name: A
Receiver Name: W
Transaction Value: 539
Transaction timestamp: 1